crown-court-contribution/src/main/resources/schemas/apiMaatCheckContributionRuleRequest.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "apiMaatCheckContributionRuleRequest.json",
+  "type": "object",
+  "title": "Calculate Contribution Request from MAAT application",
```

Comment:
Need to update the title and description.

Reply:
Updated